appveyor.yml

```diff
@@ -28,8 +32,28 @@ before_build:
   - flake8 ./scripts/
 
 build_script:
-  - cmake .  -DCMAKE_BUILD_TYPE=%CONFIG% -G "%GENERATOR%"
+  # Request `libcmt.lib` is used so our released artifacts don't dynamically
+  # link to `msvcrt.dll`
+  - cmake .  -DCMAKE_BUILD_TYPE=%CONFIG% -G "%GENERATOR%" -DMSVC_USE_LIBCMT=YES
   - cmake --build . --config %CONFIG% -- %PARALLEL_FLAG%%DEGREE_OF_PARALLELISM%
 
 test_script:
   - ctest --output-on-failure --timeout 10 -j 5
+
+before_deploy:
+  - ps: |
+        $NAME = "binaryen-${env:APPVEYOR_REPO_TAG_NAME}-${env:ARCH}-windows"
+        Move-Item -Path bin -Destination $NAME
+        7z a -ttar "${NAME}.tar" "${NAME}"
+        7z a "${NAME}.tar.gz" "${NAME}.tar"
+        Push-AppveyorArtifact "${NAME}.tar.gz"
```

Comment:
Its not more common to prefer `.zip` for windows artifacts and `.tar.??` for UNIX?

Reply:
True! The use case in https://github.com/WebAssembly/binaryen/issues/1695 was motivated by downloading precompiled binaries programmatically and it's easier to download one format vs two for us at least. I don't personally know the invocation to create a zip file, but it should be easy to make both if needed!